doc/development/05_rest_api.md

```diff
@@ -121,7 +121,6 @@ Currently the following endpoints are provided:
 
 | Method | Path                                            | Function                                                                        |
 | ---    | ---                                             | ---                                                                             |
-| GET    | /groups                                         | Redirects to root group (only works using personal token, not service accounts) |
```

Comment:
Was /groups really removed?

Reply:
Yes, I removed the endpoint `/groups` in 7794588fc656b6c7acf2f016d9c6aa301d446385 because
1. it just redirected to the path of the root group, but redirects in JSON APIs generally require more effort from the client to follow correctly
1. it only worked when using personal user tokens, it didn't work with service accounts or OAuth API access
1. API clients are mostly programmed for a fixed hitobito instance, and the root group id can easily be configured on the client if needed
1. it's easy to add back in a more consistent way once this PR is merged, if there really is demand for this endpoint

Meanwhile, retrieving a single group using the `/groups/{id}` endpoint still works of course.

Is there a need for this specific endpoint in your application? What are your thoughts?

Reply:
Thanks for the context, that's fine with me.